### DISCUSSION TITLE
Audio in downloaded files

### DISCUSSION THREAD

**Discussion by kornelijussl** [fent/node-ytdl-core#974] [Q&A]
So as I understand if you change the default format of desirable file, the would be no sound for all remaining formats such as 1080p, 720p and so on?

**Comment by TimeForANinja** [comment]
can you elaborate what you mean?

**Reply by kornelijussl** [reply]
Sure,

For exampe I was trying to adjust the configuration in the way that first priority was for the : 137, 136 itags did some check if there is such a formats and so on, and in result i've got only videos in the desirable quality but no sound with it.

![image](https://user-images.githubusercontent.com/60582843/127622867-6cc4b533-fa43-4ef1-91a6-9621f0b30b63.png)

**Reply by TimeForANinja** [reply]
yep, nearly all video qualitys better than the default come without audio (livestreams can behave different)
you can use sth like ffmpeg to merge audio and a video tracks together, though

**Reply by kornelijussl** [reply]
Alright I see, thank you for your answer. =)

**Reply by TimeForANinja** [reply]
you're welcome
if you want you can have a look at the examples - we have one for merging audio and video in js with ffmpeg
https://github.com/fent/node-ytdl-core/blob/master/example/ffmpeg.js
or you can have a look at https://www.npmjs.com/package/ytdl-core-muxer
the interface is pretty much identical to ytdl-core but it does the merging for you
unfortunately @redbrain didn't put to much time in documentation 😂
